.github/workflows/cd.yaml

```diff
@@ -1,43 +1,99 @@
-name: CD
+name: Nightly CD
```

Comment:
This should probably be "Latest Commit" or similar, and not "nightly" since its continuous